tests/prototest/test_id.golden

```diff
@@ -0,0 +1,87 @@
+// Generated from test.proto
```

Comment:
can we name these files like `test_id.fbs.golden` or something, so we know the format of the file by looking at the name?